lib/tasks/deleted_objects.rake

```diff
@@ -11,4 +11,16 @@ namespace :deleted_objects do
       sleep(0.5) unless Rails.env.test?
     end
   end
+
+  desc 'Destroy all the stale objects whose owner is also a deleted object'
+  task :destroy_the_stale_object => :environment do
```

Comment:
```suggestion
  task :destroy_stale_or_service_objects => :environment do
```
Because otherwise, it seems like it does the same as `DeletedObject.stale`, but in reality, this just means to remove the previous stale objects since the service objects won't be stored anymore.